framework/doc/content/source/materials/GenericConstantVectorFunctorMaterial.md

```diff
@@ -2,7 +2,7 @@
 
 !syntax description /Materials/GenericConstantVectorFunctorMaterial
 
-The functor version of [GenericConstantVectorFunctorMaterial.md], this can be
+The functor version of [GenericConstantFunctorMaterial.md], this can be
```

Comment:
This clearly wasn't right before but it still doesn't look right now. Guessing that `Functor` should not be in the name of the thing we are a functor version of

Reply:
oh true true must have misclicked somehow